source/dyaml/node.d

```diff
@@ -610,8 +610,15 @@ struct Node
                             " out of range. Value: " ~ to!string(temp), startMark_));
                     return temp.to!T;
                 }
+                else static if(is(typeof(T.fromString(string.init)) == T))
+                {
+                    enforce(type == NodeType.string, new NodeException(
+                                "Node stores unexpected type: " ~ text(type) ~
+                                ". Expected: " ~ typeid(string).toString(), startMark_));
+                    return T.fromString(getValue!string);
+                }
```

Comment:
We shouldn't have to modify D-YAML locally.

Reply:
Are you saying we should upstream this change before we merged this?

Reply:
Yes, but also, this change shouldn't be necessary, Configy is the one that recognize `fromString` / `this(string)`. What error did you see ?